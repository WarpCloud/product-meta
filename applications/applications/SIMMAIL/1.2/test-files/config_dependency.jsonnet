{
  application_type: "SIMMAIL",
  application_name: "simmail1",
  application_version: "1.1",
  user_config: {},
}
