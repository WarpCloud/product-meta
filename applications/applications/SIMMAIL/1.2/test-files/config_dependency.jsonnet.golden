{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "simmail1",
            "application_type": "SIMMAIL"
         },
         "configs": {
            "simmail_cpu_limit": 2,
            "simmail_cpu_request": 0.10000000000000001,
            "simmail_memory_limit": 4,
            "simmail_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "simmail",
         "name": "simmail1-simmail",
         "version": "1.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
